libafl_bolts/src/serdeany.rs

```diff
@@ -41,7 +41,7 @@ where
 
 #[cfg(feature = "stable_anymap")]
 fn type_repr_owned<T>() -> TypeRepr {
-    type_name::<T>().to_string()
+    TypeRepr::from(type_name::<T>().to_string())
```

Comment:
type_name returns a static str, so there's no need to do `to_string()` here, else we still allocate